azext_edge/edge/providers/check/common.py

```diff
@@ -49,8 +59,40 @@ class DataprocessorDestinationStageType(ListableEnum):
     reference_data = "output/refdata"
 
 
+class DataprocessorAuthenticationType(ListableEnum):
+    """
+    Data processor authentication type.
+    """
+    accessKey = "accessKey"
+    accessToken = "accessToken"
+    header = "header"
+    metadata = "metadata"
```

Comment:
header and metadata are awk for authentication type, especially when you compare them to service principal, managed identity, etc

Reply:
yep it's from the service spec, therefore will resolve it, thanks for checking!